performance/benchmarks/output/.gitignore

```diff
@@ -1,3 +0,0 @@
-*.log
```

Comment:
I know you've got a `mkdir -p` to create this if needed, but you could always also add a `.keep` file in this directory to make sure it's always around. I'm not sure if your purpose removing this file was to **avoid** checking in this directory though.

Reply:
I was just avoiding checking in that directory since there wouldn't ever be any files in there that would be checked in